2try/style.css

```diff
@@ -0,0 +1,44 @@
+*{
+    box-sizing: border-box;
+}
+.container{
+    display: grid;
+    grid-template-columns: 300px 600px 350px;
```

Comment:
maybe you could use fr and auto for grids... it would get trickier when you want to create a responsive page